apps/src/netsim/NetSimVizWire.js

```diff
@@ -192,29 +174,33 @@ NetSimVizWire.prototype.setEncodings = function(newEncodings) {
  * Kick off an animation of the wire state being set by the local viznode.
  * @param {"0"|"1"} newState
  */
-NetSimVizWire.prototype.animateSetState = function(newState) {
+NetSimVizWire.prototype.animateSetState = function(newState, fromRemote) {
```

Comment:
Do you need the `fromRemote` parameter anymore?  I see that you're still using it below, but I don't see where you've modified any calls to pass it.